vpdq/cpp/regtest.py

```diff
@@ -73,90 +81,91 @@ def get_argparse() -> argparse.ArgumentParser:
 
 
 def dir_path(string):
-    if os.path.isdir(string):
+    if Path(string).is_dir():
         return string
-    else:
-        raise argparse.ArgumentTypeError(f"readable_dir: {string} is not a valid path")
+    raise argparse.ArgumentTypeError(
+        f"readable_dir: {string} is not a valid directory path"
+    )
 
 
 def main():
+    hashVideoExecutable = EXEC_DIR / "vpdq-hash-video"
+    matchHashesExecutable = EXEC_DIR / "match-hashes-byline"
+
+    if not hashVideoExecutable.exists() or not matchHashesExecutable.exists():
+        print(
+            "Error: Hashing executable/s not found. Build vpdq before running regtest."
+        )
+        sys.exit(1)
+
     ap = get_argparse()
     args = ap.parse_args()
-    inputVideoFolder = args.inputVideoFolder
-    outputHashFolder = args.outputHashFolder
+    inputVideoFolder = Path(args.inputVideoFolder)
```

Comment:
nit: Another approach we can sue here is to use argparse's `type=Path` to force it to be parsed there. We could go a step further and also check things like existence and throw exceptions if they aren't perfect, though I'm fine not thoroughly validating given the nature of this script. 

Both click and argparse modules have pre-built filetypes, but we chose when building this repo to take a draconian approach to not using any external dependencies if we didn't need to, though click is quite nice.

Reply:
I believe what you're explaining is already done through the `type=dir_path` and `dir_path` function. I'll add a check for existence for the input folder. The output hash folder should not exist before running, but this won't matter if I use tempfile.